docs/unified-test-documentation/dasharo-compatibility/31S-sata-led-and-pc-speaker-errror-indication.md

```diff
@@ -0,0 +1,35 @@
+# Dasharo Compatibility: SATA LED and PC speaker error indication
```

Comment:
@miczyg1 we should also add this new test suite to `mkdocs` file. Without this, the test suite should not be visualized in the menu on the left of the screen.

Reply:
Added to `mkdocs.yml`